### ISSUE TITLE
URI deferenceing

### ISSUE THREAD

**Issue by bertvannuffelen** [w3c/vc-data-model#764]
In https://www.w3.org/2018/credentials/v1

jsonld mappings are made to cred:credentialStatus which corresponds with the URI 
https://www.w3.org/2018/credentials#credentialStatus.

This URI results in a 404.

kr,

Bert

**Comment by kdenhartog** [comment]
Yeah unfortunately the IRIs within the context don't actually link to definitions of the properties. This is a mistake that's been recently discussed on the mailing list about how to fix. For now the definition for this term can be found here: https://www.w3.org/TR/vc-data-model/#status

**Comment by kdenhartog** [comment]
Seeing as this would be a change to the context id's which would constitute breaking all of the currently issued VCs, I'm going to mark this as a V2 concern that will need to be addressed then.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2022-08-03-vcwg#section6-2) on 2022-08-03

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 6.2. URI dereferencing (issue vc-data-model#764)

_See github issue [vc-data-model#764](https://github.com/w3c/vc-data-model/issues/764)._




> *Orie Steele:* This is core data model thing..

**Manu Sporny:** My understanding is that this is about the base credentials context; dereferencing that feels like a data model thing, not a data integrity thing..  

**Brent Zundel:** Anyone have any comments or concerns with keeping it as a data model issue rather than data integrity issue?.  

</details>

**Comment by msporny** [comment]
These links now exist: https://w3c.github.io/vc-data-model/vocab/credentials/v2/vocabulary.html#credentialStatus

We just need to update the vocabulary redirects to point to the new locations. This issue is ready for a PR that updates the vocabulary URLs to the updated VC vocabulary. Assigning to @iherman since he will be the one to do that once we have approval from the VCWG.

**Comment by OR13** [comment]
related issue: https://github.com/w3c/vc-data-model/issues/990#issuecomment-1342312804

**Comment by iherman** [comment]
@bertvannuffelen I believe the issue is now solved. Could you check and close the issue if you are fine?
